invenio_files_rest/__init__.py

```diff
@@ -6,7 +6,743 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Files download/upload REST API similar to S3 for Invenio."""
+
+r"""REST API for Files similar to S3 for Invenio.
+
+Invenio-Files-REST is a core component of Invenio which provides configurable
+REST APIs for uploading, serving, downloading and deleting files. It works in
+combination with `Invenio-Records <https://invenio-records.readthedocs.io>`_
+and `Invenio-Records-Files <https://invenio-records-files.readthedocs.io>`_
+but also as a standalone module.
+
+The module can be configured with different storage backends, and provides
+features such as:
+
+- A robust REST API.
+- Configurable storage backends with the ability to build your very own.
+- Highly customizable permission mechanism for access-control.
+- Secure file handling.
+- Integrity check mechanism.
+- Supports for large file uploads and multipart upload.
+- Signals for system events.
+
+The REST API follows best practices and supports, e.g.:
```

Comment:
Because of the comma after supports it sounds like supports was meant to be a plural form of a noun- and for me this doesn't really fit in the sentence. I would either remove the comma so it's clear that supports is a verb, or even better phrase it somehow along the lines of "... and provides following features:"